### ISSUE TITLE
conversion to ipf doesn't work

### ISSUE THREAD

**Issue by BitBlit88** [jfdelnero/HxCFloppyEmulator#3]
When I tried to convert any raw 1.44MB DOS image to ipf, a 17KB file is created that contains no data.
For example:

Take a FreeDOS disk image from: https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/distributions/1.3/official/FD13-FloppyEdition.zip)

And try to convert with the command:

hxcfe.exe -finput:x86BOOT.img  -foutput:x86BOOT.ipf  -conv:SPS_IPF

An invalid ipf file is created.

**Comment by jfdelnero** [comment]
Yes i confirm that the IPF writer is currently very far to be finished.

**Comment by jfdelnero** [comment]
Code source :
https://github.com/jfdelnero/HxCFloppyEmulator/blob/main/libhxcfe/sources/loaders/ipf_loader/ipf_writer.c
